orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -229,6 +244,13 @@ class RedisQueue(
       } else {
         try {
           val message = convertToMessage(json, mapper)
+
+          // TODO: AttemptsAttribute could replace `attemptsKey`
+          message.setAttribute(
```

Comment:
Heavy +1. I'd like to move basically everything metadata-y to attribute classes.